### DISCUSSION TITLE
部分npm包在verdaccio容器中存在也可以安装，但是web界面不显示也搜不到这个npm包

### DISCUSSION THREAD

**Discussion by luban-130** [verdaccio/verdaccio#4725] [Q&A]
**Your Environment**
 * **verdaccio version**: 5.27.0
 * **node version** :16.18.1
 * **package manager**: 8.19.2
 * **os**:linux
 * **platform**: docker

**Describe the bug**
[部分npm包在verdaccio容器中存在也可以安装，但是web界面不显示也搜不到这个npm包
![verdaccio容器](https://github.com/user-attachments/assets/83c10ba6-675b-4181-a210-d9b748babb64)

![verdaccio界面不显示](https://github.com/user-attachments/assets/40406140-324c-4f49-a4ce-af3559c56b23)

**Comment by juanpicado** [comment]
@luban-130 (feel free to translate my words) Not all packages are displayed on the User Interface, only the private ones, the ones you have published, this is by design.  The search could be not perfect but I'd suggest you upgrade to the latest first.

**Reply by luban-130** [reply]
> @luban-130 (feel free to translate my words) Not all packages are displayed on the User Interface, only the private ones, the ones you have published, this is by design. The search could be not perfect but I'd suggest you upgrade to the latest first.

一般都是先用npm pack命令从公网上拉取，再用npm publish发布到verdaccio中的，某些包依然在界面不显示

**Reply by juanpicado** [reply]
I would indeed check the https://verdaccio.org/docs/configuration#.verdaccio-db database, if the packages are there then should be shown otherwise could be different other reasons, check the logs to collect more info.
